### PR TITLE
UI Polishment

### PR DESCRIPTION
**Pull request by rafaelmardojai**
Captura de pantalla de 2019-08-02 09-26-35
Captura de pantalla de 2019-08-02 09-27-21

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
What do you think about adding a separator?
Captura de pantalla de 2019-08-02 10-34-12

**Comment by sonnyp** [comment]
Hey thanks.

This PR looks weird, it moves/removes code that I just commited.
Is it because of back rebase and bad luck that I submitted those changes https://github.com/sonnyp/gigagram/commit/49ab31063c527dc4a38efe79c6f4b5e3ffd678c3 right before your PR?

Next time please manifest yourself on an issue so I know someone is already working on it :)

Please revert changes that are not adding anything new.

**Comment by sonnyp** [comment]
@rafaelmardojai nah it's not a common pattern and I think following Epiphany on this one is not a bad idea see https://github.com/sonnyp/gigagram/issues/20

File: src/main.js [hunk 1]
```diff
@@ -143,6 +143,7 @@
         logo_icon_name: "re.sonny.gigagram",
       });
       aboutDialog.add_credit_section("Contributors", [
+        "Rafael Mardojai CM",
```
Comment (sonnyp) [inline src/main.js]: :+1:

File: data/menu.xml.ui [hunk 2]
```diff
@@ -56,14 +53,18 @@
...
+            <property name="height_request">30</property>
+            <property name="visible">True</property>            
+            <property name="text" translatable="yes">Tabs position</property>
+            <property name="menu_name">menu</property>
             <property name="inverted">True</property>
             <property name="centered">True</property>
-            <!-- <property name="action-name">app.tabsPosition</property> -->
-            <!-- <property name="action-target">"top"</property> -->
-            <property name="text" translatable="yes">Tabs position</property>
-            <property name="menu-name">main</property>
```
Comment (sonnyp) [inline data/menu.xml.ui]: this is the correct property, not `menu_name`

File: data/menu.xml.ui [hunk 3]
```diff
@@ -2,52 +2,49 @@
...
             <property name="visible">True</property>
             <property name="text" translatable="yes">Quit</property>
             <property name="action-name">app.quit</property>
           </object>
-        </child> -->
+        </child>
       </object>
+      <packing>
+        <property name="submenu">menu</property>
+      </packing>
```
Comment (sonnyp) [inline data/menu.xml.ui]: I don't think this does anything

File: data/menu.xml.ui [hunk 4]
```diff
@@ -2,52 +2,49 @@
...
         <child>
           <object class="GtkModelButton">
             <property name="visible">True</property>
-            <property name="text" translatable="yes">About Gigagram</property>
+            <property name="text" translatable="yes">About</property>
             <property name="action-name">app.about</property>
           </object>
         </child>
-        <!-- Probably no need for this for now since closing the window achieves the same -->
-        <!-- <child>
```
Comment (sonnyp) [inline data/menu.xml.ui]: do not remove code/comments please

File: data/menu.xml.ui [hunk 5]
```diff
@@ -2,52 +2,49 @@
...
-            <property name="text" translatable="yes">Keyboard Shortcuts</property>
+            <property name="text" translatable="yes">Shortcuts</property>
             <property name="action-name">app.shortcuts</property>
           </object>
         </child>
         <child>
           <object class="GtkModelButton">
             <property name="visible">True</property>
-            <property name="text" translatable="yes">About Gigagram</property>
+            <property name="text" translatable="yes">About</property>
```
Comment (sonnyp) [inline data/menu.xml.ui]: no

File: data/menu.xml.ui [hunk 6]
```diff
@@ -2,52 +2,49 @@
...
-          <object class="GtkSeparatorMenuItem">
-            <property name="visible">True</property>
+            <property name="text" translatable="yes">Tabs position</property>
           </object>
         </child>
         <child>
           <object class="GtkModelButton">
             <property name="visible">True</property>
-            <property name="text" translatable="yes">Keyboard Shortcuts</property>
+            <property name="text" translatable="yes">Shortcuts</property>
```
Comment (sonnyp) [inline data/menu.xml.ui]: no

File: data/menu.xml.ui [hunk 7]
```diff
@@ -2,52 +2,49 @@
...
         <child>
           <object class="GtkModelButton">
             <property name="visible">True</property>
-            <property name="text" translatable="yes">Tabs position</property>
             <property name="menu-name">tabs-position</property>
-          </object>
-        </child>
-        <child>
-          <object class="GtkSeparatorMenuItem">
-            <property name="visible">True</property>
```
Comment (sonnyp) [inline data/menu.xml.ui]: keep the code

File: data/menu.xml.ui [hunk 8]
```diff
@@ -2,52 +2,49 @@
...
+      <object class="GtkBox" id="menu">
         <property name="visible">True</property>
         <property name="margin">10</property>
         <property name="orientation">1</property>
         <child>
           <object class="GtkModelButton">
             <property name="visible">True</property>
-            <property name="text" translatable="yes">New application</property>
             <property name="action-name">app.newApplication</property>
+            <property name="text" translatable="yes">New application</property>
```
Comment (sonnyp) [inline data/menu.xml.ui]: please avoid moving code around for no reason, it makes reviewing harder

File: data/menu.xml.ui [hunk 9]
```diff
@@ -2,52 +2,49 @@
 <interface>
   <object class="GtkPopoverMenu" id="app-menu">
     <child>
-      <object class="GtkBox">
+      <object class="GtkBox" id="menu">
```
Comment (sonnyp) [inline data/menu.xml.ui]: no need

File: src/header.js [hunk 10]
```diff
@@ -53,40 +52,39 @@
...
-    reloadButton.connect("clicked", onReload);
-
-    const addTabButton = Button.new_from_icon_name(
-      "tab-new-symbolic",
-      IconSize.BUTTON
-    );
-    buttonBox.add(addTabButton);
-    addTabButton.set_always_show_image(true);
-    addTabButton.connect("clicked", onAddTab);
-
```
Comment (sonnyp) [inline src/header.js]: please revert the whole file

File: data/menu.xml.ui [hunk 11]
```diff
@@ -56,14 +53,18 @@
...
-            <!-- <property name="action-name">app.tabsPosition</property> -->
-            <!-- <property name="action-target">"top"</property> -->
-            <property name="text" translatable="yes">Tabs position</property>
-            <property name="menu-name">main</property>
           </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+            <property name="position">0</property>
+          </packing>
```
Comment (sonnyp) [inline data/menu.xml.ui]: Could you provide a screenshot before/after to make reviewing easier? Looks good though

**Comment by rafaelmardojai** [comment]
Sorry, i didn't see that issue. Closing.

**Comment by rafaelmardojai** [comment]
My clone was before 49ab310, that's the problem.

**Comment by sonnyp** [comment]
@rafaelmardojai bad luck :/ feel free to open a PR/issue if you see more UI improvements to make. Make sure to follow GTK UI/style.

**Pull request closed without merging**